### ISSUE TITLE
AWS - mas install - 1 node(s) did not have enough free storage.

### ISSUE THREAD

**Issue by JedrekWie** [ibm-mas-manage/sno#70]
I followed the AWS SNO provisioning instructions from https://ibm-mas-manage.github.io/sno/
New `m5.4xlarge` instance has been successfully provisioned with EBS storage of size 120 GB.
Unfortunately `mas install` pipeline failed to execute at fairly early stage due to insufficient storage (see screenshots below).

![image](https://user-images.githubusercontent.com/24481718/215993117-28fc4ccb-2b6f-4c1a-a489-2830f503ea1d.png)
![image](https://user-images.githubusercontent.com/24481718/215993153-9a421202-7faf-4b2b-8a3e-43c9bb546476.png)

Is there a way to control the storage size od the AWS instance at the time of provisioning?

I could try to extend the storage manually but I actually have troubles finding details how to connect to the instance using SSH and issue few commands to make use of extended EBS volume size...

**Comment by dirkhuppertz** [comment]
I had the same error.
It works if you skip the chapter "Storage Class" and choose the "AWS gp2" storage.
![image](https://user-images.githubusercontent.com/120086785/216011054-f1deb9dc-996b-4122-8c62-e1a963a7dc09.png)

**Comment by JedrekWie** [comment]
Thanks @dirkhuppertz for sharing. I was wondering whether to go for `gp2` or lvm but wanted to keep everything local on this SNO instance.
Could you tell me how does the instance look if you use `gp2` storage class? Do you get new EBS volumes automatically provisioned and attached to the OC instance?
My goal is to reduce the footprint of the SNO in terms of new AWS configuration objects as much as possible so that it's manageable to keep control over potentially multiple different SNO-instances.

**Comment by dirkhuppertz** [comment]
Yes, in AWS my EC2 instance now has 16 new volumes. As you can see on the attachment timestamp, they are all created by the script during the installation process this morning.
![image](https://user-images.githubusercontent.com/120086785/216077194-e169ed22-0d04-421f-8097-fbb116966851.png)
